wallet/wallet.sol

```diff
@@ -363,16 +362,56 @@ contract Wallet is multisig, multiowned, daylimit {
             return true;
         }
     }
-    
+
+    // Gets the number of pending transactions
+    function numPendingTransactions() returns (uint) {
+        var pendingTransactionsCount = 0;
+        // Use m_pendingIndex.length to get all hashes, then count how many of the
+        // operations are transactions, because we don't want to store hashes twice
+        // and this is a local call anyway
+        for (uint i = 0; i < m_pendingIndex.length; i++) {
+            if (isPendingTransaction(i)) {
+              pendingTransactionsCount++;
```

Comment:
Same as above, `pendingTransactionsCount` would be an `uint8`. Can it be guaranteed this will never overflow?